scripts/add_connector.sh

```diff
@@ -54,40 +54,129 @@ previous_connector_camelcase="$(tr '[:lower:]' '[:upper:]' <<< ${previous_connec
+sed -i'' -e "/pub ${previous_connector}: Option<ConnectorTomlConfig>,/a\\
+    pub ${payment_gateway}: Option<ConnectorTomlConfig>,
+" crates/connector_configs/src/connector.rs
+
 sed -i'' -e "/mod utils;/ s/mod utils;/mod ${payment_gateway};\nmod utils;/" crates/router/tests/connectors/main.rs
 sed -i'' -e "s/^default_imp_for_new_connector_integration_payouts!(/default_imp_for_new_connector_integration_payouts!(\n    connector::${payment_gateway_camelcase},/" crates/router/src/core/payments/connector_integration_v2_impls.rs
 sed -i'' -e "s/^default_imp_for_new_connector_integration_frm!(/default_imp_for_new_connector_integration_frm!(\n    connector::${payment_gateway_camelcase},/" crates/router/src/core/payments/connector_integration_v2_impls.rs
 sed -i'' -e "s/^default_imp_for_new_connector_integration_connector_authentication!(/default_imp_for_new_connector_integration_connector_authentication!(\n    connector::${payment_gateway_camelcase},/" crates/router/src/core/payments/connector_integration_v2_impls.rs
-sed -i'' -e "s/\(pub enum Connector {\)/\1\n\t${payment_gateway_camelcase},/" crates/common_enums/src/connector_enums.rs
-sed -i'' -e "/match self {/ s/match self {/match self {\n            | Self::${payment_gateway_camelcase}/" crates/common_enums/src/connector_enums.rs
-sed -i'' -e "/match routable_connector {/ s/match routable_connector {/match routable_connector {\n            RoutableConnectors::${payment_gateway_camelcase} => Self::${payment_gateway_camelcase},/" crates/common_enums/src/connector_enums.rs
-sed -i'' -e "/match self.connector_name {/a\\
-            api_enums::Connector::${payment_gateway_camelcase} => {\\
-                ${payment_gateway}::transformers::${payment_gateway_camelcase}AuthType::try_from(self.auth_type)?;\\
-                Ok(())\\
-            },\\
-" crates/router/src/core/admin.rs
+
 
 
 # Remove temporary files created in above step
 rm $conn.rs-e $src/types/api.rs-e $src/configs/settings.rs-e config/development.toml-e config/docker_compose.toml-e config/config.example.toml-e loadtest/config/development.toml-e crates/api_models/src/connector_enums.rs-e crates/euclid/src/enums.rs-e crates/api_models/src/routing.rs-e $src/core/payments/flows.rs-e crates/common_enums/src/connector_enums.rs-e $src/types/transformers.rs-e $src/core/admin.rs-e crates/hyperswitch_connectors/src/default_implementations.rs-e crates/hyperswitch_connectors/src/default_implementations_v2.rs-e crates/hyperswitch_interfaces/src/configs.rs-e $src/connector.rs-e config/deployments/integration_test.toml-e config/deployments/production.toml-e config/deployments/sandbox.toml-e temp crates/connector_configs/src/connector.rs-e crates/router/tests/connectors/main.rs-e crates/router/src/core/payments/connector_integration_v2_impls.rs-e
+
+sed -i'' -e "/pub ${previous_connector}: ConnectorParams,/a\\
```

Comment:
Isn't this going to generate more temporary files, but we' have a step before this which says "Remove temporary files created in above step".

Can we move this line above the `rm` command?